Makefile

```diff
@@ -39,7 +39,7 @@ KONG_NETTLE_VERSION ?= `grep KONG_NETTLE_VERSION $(KONG_SOURCE_LOCATION)/.requir
 KONG_NGINX_MODULE ?= `grep KONG_NGINX_MODULE $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 OPENRESTY_PATCHES ?= 1
 LIBYAML_VERSION ?= 0.2.3
-DOCKER_KONG_VERSION ?= 'master'
+DOCKER_KONG_VERSION ?= 'fix/remove-dev-packages'
```

Comment:
merge https://github.com/Kong/kong-distributions/pull/462 then remove and merge this PR